romana_setup.md

```diff
@@ -0,0 +1,117 @@
+# The `romana-setup` tool
+
+The `romana-setup` tool is used to provision and install Romana using the selected installation environment, linux distribution and stack type.
+
+## Usage Summary
+
+```
+romana-setup: executes the ansible playbooks for installing romana
+Usage: romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] [action]
+       romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] <action> [ansible-options]
+       romana-setup <h|--help>
+
+Stack Name:  user-defined stack name (default: $USER)
```

Comment:
The parameter 'name' does not have an obvious relationship to 'Stack name'.

Maybe change the 'name' parameter to 'stackname' again, or let this line start with 'Name'.